communication/marketing-team/CHARTER.md

```diff
@@ -0,0 +1,45 @@
+# Kubernetes Upstream Marketing Team Charter
+
+## Summary 
+
+The mission of the Kubernetes Upstream Marketing Team is to elevate the hard work being done throughout the Kubernetes community by its contributors. Promoting the work and helping fill the needs of the Kubernetes community is the driving force behind the Kubernetes Upstream Marketing Team. The goal of this work is to improve the overall sustainability of the project by giving or elevating information to contributors so they stay well informed.
+
+## Ethos/Vision
+
+This group's work is in support of the upstream Kubernetes community (the open source project). As such it will adhere to the [Kubernetes Community Values](https://github.com/kubernetes/community/blob/master/values.md#kubernetes-community-values). The core value we feel strongest about is [Community over product or company](https://github.com/kubernetes/community/blob/master/values.md#community-over-product-or-company). The makeup of this team will be from all walks of life including vendors. The work of the Kubernetes community should always be prioritized over vendor created works. Kubernetes Upstream Marketing Team members are expected to act in the interest of the community and not their employers.
+
+## Goals
+
+The goal of the Kubernetes Upstream Marketing Team is to be of service to the community. It will act to facilitate conversations across the Kubernetes community. It will also act as a voice of the community to broaden the reach and effectiveness of community communications. Facilitating those communications is a critical function of this team.
+
+## Out of Scope
+
+This team operates independently from Cloud Native Computing Foundation (CNCF) operations. The Team works with CNCF as needed but it exists within the Kubernetes governance model and is not a part or extension of CNCF Marketing. As such, this is not the team to reach out to about official CNCF Events and/or KubeCon. #kubecon on slack can be a helpful resource there. This team can assist in some ways with other CNCF projects but, the first priority of the team is Kubernetes.
+
+## Governance
+
+TODO
+
+### Roles
+
+* Internal Comms
+* Social Media
+* Web/Blog
+
+All roles should have at least one shadow. All leads should actively recruit shadows for the entire team.
+
+TODO
+
+### Decision Making
+
+-lazy consensus (policy)
+
+-leads make calls on smaller editorial and transactional (example: twitter)
```

Comment:
Leads are encouraged to seek guidance from @kubernetes/sig-contributor-experience and/or other community members/teams as appropriate.